docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -138,6 +170,16 @@ View the state of any inbox to see the identities, installations, and other info
 
 :::code-group
 
+```tsx [Browser]
+const state = await client.preferences.inboxState(true)
+const states = await client.preferences.inboxStateFromInboxIds(true, [inboxId, inboxId])
```

Comment:
the arguments are reversed, should be:

```
const states = await client.preferences.inboxStateFromInboxIds([inboxId, inboxId], true)
```

the second argument is optional and refreshes the state from the network.